packages/synapse-react-client/src/components/styled/LightTooltip.tsx

```diff
@@ -8,7 +14,7 @@ export const LightTooltip: StyledComponent<TooltipProps> = styled(
   ),
 )(({ theme }) => ({
   [`& .${tooltipClasses.tooltip}`]: {
-    backgroundColor: theme.palette.common.white,
+    backgroundColor: theme.palette.background.paper,
```

Comment:
In general trying to use the `background` palette instead of hard-coding `white`